### PR TITLE
CLDR-15034 kbd: add iso and abnt2 keyboards

### PR DESCRIPTION
**Pull request by srl295**
CLDR-15034

- [ ] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

File: keyboards/3.0/pt-BR-k0-abnt2.xml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+<!--
+	TODO: keyman: HINT 100008: Locale 'pt-BR-t-k0-abnt2' is not minimal or correctly formatted and should be 'pt-t-k0-abnt2'
+-->
+<keyboard locale="pt-BR-t-k0-abnt2" conformsTo="techpreview">
+	<locales>
+		<locale id="pt" />
+	</locales>
+	<version number="0.0.0" />
+	<names>
+		<name value="Portugese (ABNT2)" />
```
Comment (mcdurdin) [inline keyboards/3.0/pt-BR-k0-abnt2.xml:12]:
```suggestion
		<name value="Portuguese (ABNT2)" />
```

Do we want to say `Portuguese (Brazil) (ABNT2)`?

File: keyboards/test/ja-Latn-test.xml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE keyboardTest SYSTEM "../dtd/ldmlKeyboardTest.dtd">
+<keyboardTest conformsTo="techpreview">
+	<info keyboard="ja-Latn.xml" author="Team Keyboard" name="ja-Latn-test" />
+	<repertoire name="latn-repertoire"
+		chars="[a-z A-Z 0-9 !\u0022#$%\u0026\[\]\{\}=\-|¥~\^_\u0020\u003c>,./?`@\+\*]" type="simple" />
```
Comment (mcdurdin) [inline keyboards/test/ja-Latn-test.xml:6]: Unsure about spaces between `a-z` and `A-Z` and `0-9` etc here.

Comment (srl295) [reply keyboards/test/ja-Latn-test.xml:6]: it's just for possible clarity.  Whitespace is not significant here.

**Comment by srl295** [comment]
like to merge this, layout description will be a separate PR

File: keyboards/3.0/pt-k0-abnt2.xml [hunk 3]
```diff
@@ -1,15 +1,12 @@
...
--->
-<keyboard locale="pt-BR-t-k0-abnt2" conformsTo="techpreview">
+<keyboard locale="pt-t-k0-abnt2" conformsTo="techpreview">
 	<locales>
 		<locale id="pt" />
 	</locales>
 	<version number="0.0.0" />
 	<names>
-		<name value="Portugese (ABNT2)" />
+		<name value="Portugese (Brazil) (ABNT2)" />
```
Comment (mcdurdin) [inline keyboards/3.0/pt-k0-abnt2.xml:9]:
```suggestion
		<name value="Portuguese (Brazil) (ABNT2)" />
```

**Review by mcdurdin — APPROVED** [review]

**Pull request merged**